File: airline_code_lookup/v1/get/node/airline_code_lookup.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// What's the airline name for the IATA code BA?
amadeus.referenceData.airlines.get({
  airlineCodes: 'BA'
}).then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});

File: airline_code_lookup/v1/get/python/airline_code_lookup.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client()

try:
    '''
    What's the airline name for the IATA code BA?
    '''
    response = amadeus.reference_data.airlines.get(airlineCodes='BA')
    # print(response.data)
except ResponseError as error:
    raise error
